Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -175,6 +175,23 @@
   icon: Objects/Misc/id_cards.rsi/default.png
   price: 3
 
+- type: uplinkListing
+  id: UplinkHolster
+  category: Utility
+  itemId: ClothingBeltSyndieHolster
+  listingName: Syndicate Shoulder Holster
+  description: A deep shoulder holster capable of holding many types of ballistics.
+  icon: /Textures/Objects/Tools/emag.rsi/icon.png
```

Comment:
This icon needs to be changed to the icon of the holster, otherwise it's going to show the image of an emag in the uplink listing.

Reply:
> This icon needs to be changed to the icon of the holster, otherwise it's going to show the image of an emag in the uplink listing.

fixed this